src/main/kotlin/app/revanced/cli/command/PatchCommand.kt

```diff
@@ -254,6 +255,10 @@ internal object PatchCommand : Runnable {
                     signer
                 )
             )
+            else {
+                outputFilePath.toPath().deleteIfExists()
+                tempFile.copyTo(outputFilePath)
```

Comment:
How about instead moving the file? Should reduce duplicating the file.

Reply:
What I mean't that at one point in time more space is consumed for no reason, moving should solve this.